account/identity.go

```diff
@@ -140,10 +142,18 @@ func (m *GormIdentityRepository) Create(ctx context.Context, model *Identity) er
 	}
 	err := m.db.Create(model).Error
 	if err != nil {
-		goa.LogError(ctx, "error adding Identity", "error", err.Error())
+		log.LogError(ctx, map[string]interface{}{
```

Comment:
I'm not sure but I think it is also considered bad practice to log the error here if we don't handle it? All we do is returning the error to the caller. In that case we should not log it here at all. This PR should probably not blindly replace old log messages but instead make sure that we only log in situations where we deal with the error.